pkg/epp/backend/vllm/metrics.go

```diff
@@ -63,8 +53,6 @@ func (p *PodMetricsClientImpl) FetchMetrics(
 	logger := log.FromContext(ctx)
 	loggerDefault := logger.V(logutil.DEFAULT)
 
-	// Currently the metrics endpoint is hard-coded, which works with vLLM.
-	// TODO(https://github.com/kubernetes-sigs/gateway-api-inference-extension/issues/16): Consume this from InferencePool config.
 	url := "http://" + existing.Address + ":" + strconv.Itoa(int(port)) + "/metrics"
```

Comment:
Pls keep the TODO. We should make the metrics port and path configurable, but can be a followup.